kart/status.py

```diff
@@ -22,7 +23,13 @@
     type=click.Choice(["text", "json"]),
     default="text",
 )
-def status(ctx, output_format):
+
+@click.option(
+    "--list-untracked-tables",
```

Comment:
This option has slightly weird behaviour - 
- `kart status` gathers the status of the kart repo and prints it as text
- `kart status --list-untracked-tables` gathers the status of the kart repo, then discards it, and lists the untracked tables
- `kart status --list-untracked-tables -o json` gathers the status of the kart repo and prints it as json, the list-untracked-tables option has no effect

Ideally, since this is an option and not a new command, we would probably want to *add* to the output, both for text and JSON output - ie, running `kart status --list-untracked-tables` would show you the kart status plus the untracked tables. You might feel that this is beyond the scope of this PR though - for now I will be happy if the list_untracked_tables option is checked first, so that we don't do the unnecessary work of generating the rest of the status and then discarding it